### PR TITLE
Docs 4309 correct active directory articles

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR corrects both the SCALE and CORE UI reference articles for Active Directory screen setting on Allow Trusted Domains.
It also breaks out the SCALE Directory Services tutorials into individual articles for each directory service.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigKerberosSCALE.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+* *DES-CBC-MD5* adheres more closely to the MIT implementation and is backward compatible.
+* *RC4-HMAC-NT* uses 128-bit encryption.
+* *AES256-SHA1* uses AES256-CTS-HMAC-SHA1-96 encryption.
+* *AES128-SHA1* uses AES128-CTS-HMAC-SHA1-96 encryption.
+
+#### Adding the Windows Keytab to TrueNAS
+
+After generating the keytab, go back to **Directory Services** in TrueNAS and click **Add** in the **Kerberos Keytab** window to add it to TrueNAS.
+
+To make AD use the keytab, click **Settings** in the **Active Directory** window and select it using the **Kerberos Principal** drop-down.
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigKerberosSCALE.md:83]:
```suggestion
To make AD use the keytab, click **Settings** in the **Active Directory** window and select it using the **Kerberos Principal** dropdown list.
```

File: content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigKerberosSCALE.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+When using a keytab with AD, ensure the keytab **username** and **userpass** match the **Domain Account Name** and **Domain Account Password**.
+
+To make LDAP use a keytab principal, click **Settings** in the **LDAP** window and select the keytab using the **Kerberos Principal** drop-down.
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigKerberosSCALE.md:87]:
```suggestion
To make LDAP use a keytab principal, click **Settings** in the **LDAP** window and select the keytab using the **Kerberos Principal** dropdown list.
```

File: content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigLDAPSCALE.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+![LDAPSCALE](/images/SCALE/LDAPSCALE.png "LDAP Options")
+
+Enter your LDAP server hostname, then enter your LDAP server Base and Bind domain names and the bind password. Check the **Enable** box to activate the server, then click **Save**.
+
+To further modify the LDAP configuration, click **Advanced Options**. See the [LDAP UI Reference article]({{< relref "/content/SCALE/SCALEUIReference/Credentials/DirectoryServices/LDAP.md" >}}) for details about advanced settings.
+
+![LDAPAdvancedSCALE](/images/SCALE/LDAPAdvancedSCALE.png "LDAP Advanced Options")
+
+{{< taglist tag="scaleldap" limit="10" >}}
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigLDAPSCALE.md:31]: Not listing any related content?

File: content/SCALE/SCALETutorials/Credentials/DirectoryServices/_index.md [hunk 4]
```diff
@@ -9,266 +9,20 @@ aliases:
...
-Go to **Credentials > Directory Services** and click **Configure LDAP**.
-
-![LDAPSCALE](/images/SCALE/LDAPSCALE.png "LDAP Options")
-
-Enter your LDAP server hostname, then enter your LDAP server Base and Bind domain names and the bind password. Check the **Enable** box to activate the server, then click **Save**.
-
-To further modify the LDAP configuration, click **Advanced Options**. See the [LDAP UI Reference article]({{< relref "/content/SCALE/SCALEUIReference/Credentials/DirectoryServices/LDAP.md" >}}) for details about advanced settings.
-
-![LDAPAdvancedSCALE](/images/SCALE/LDAPAdvancedSCALE.png "LDAP Advanced Options")
+To view Idmap and Kerberos Services, click **Show** next to **Advanced Settings**.
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/Credentials/DirectoryServices/_index.md:19]: Should this information be part of the _index page?

**Review by clairewrtr — CHANGES_REQUESTED** [review]
Looks good, just a few questions.

File: content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigLDAPSCALE.md (hunk 3 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigLDAPSCALE.md:31]: For some reason the shortcode is not working. I do need to add the LDAP screen article but that is not part of this PR, it is in a DOCS ticket in the unreviewed list of tickets.

File: content/SCALE/SCALETutorials/Credentials/DirectoryServices/_index.md (hunk 4 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Credentials/DirectoryServices/_index.md:19]: Yes, I decided to leave that in the index page as it applied to both primary directory services. I've done this in a few articles in other sections.

**Review by clairewrtr — APPROVED** [review]
Looks good.

**Pull request merged**